config/src/test/resources/valid_config_with_quorum_config.json

```diff
@@ -0,0 +1,35 @@
+{
+  "chainId": 4,
+  "eip150Block": 2,
+  "homesteadBlock": 1,
+  "eip150Hash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+  "eip155Block": 3,
+  "eip158Block": 3,
+  "byzantiumBlock": 1035301,
+  "isQuorum": true,
+  "qip714block": 99999,
```

Comment:
nit: can the B be capital?  `qip714Block`. or is it lower case in Quorum configs?  The code that parses the config drops all the cases anyway, almost entirely a style thing.

Reply:
If our code doesn't care about casing it is ok. I'll change it :)

Reply:
I've checked and it looks like our code doesn't work with any casing. I'll keep it matching Quorum's casing for now as I've seen some setups where we are using the same genesis file for both (with a few repeated properties, but I don't want to add another one to the list of different properties...).

I'm happy to discuss this further and change it later if we think it makes sense.

Reply:
Yes, keeping copy/paste compatible with Quorum configs would be a very nice thing.